permissions/plugin-ansible-tower.yml

```
@@ -7,4 +7,4 @@ paths:
   - "org/jenkins-ci/plugins/ansible-tower"
 developers:
   - "johnwestcottiv"
-  - "micr0zz" 
+  - "micr0zz"
```

This needs to be the Jira and Artifactory account, not the GitHub account.
```suggestion
  - "nupinto"
```

---

Don't forget to login as user `nupinto` to 

* https://repo.jenkins-ci.org
* https://issues.jenkins.io